src/test/java/teammates/test/pageobjects/InstructorCommentsPage.java

```diff
@@ -208,6 +210,41 @@ public void clickResponseCommentDelete(int sessionIdx, int questionIdx, int resp
         waitForPageToLoad();
     }
     
+    /**
+     * Clicks all the headings of the comment panel to either expand / collapse the panel body.
+     */
+    public void clickAllCommentsPanelHeading() {
+        for (WebElement e : browser.driver.findElements(By.cssSelector("div[id^='panel_display-']"))) {
+            e.findElement(By.cssSelector(".panel-heading")).click();
+        }
+    }
+    
+    /**
+     * Verify if the body of all the comment panel is indeed expanded / collapsed after its corresponding heading is clicked.
+     * @param visible true to verify if the comment panel body is expanded, otherwise false.
+     * @return true if all comment panel body is being expanded, otherwise false.
+     */
```

Comment:
Ideally, waiting for animation should be a separate method. Otherwise this method ends up doing multiple things.